src/java.base/share/classes/sun/nio/ch/ServerSocketChannelImpl.java

```diff
@@ -749,9 +729,7 @@ public String toString() {
                 if (addr == null) {
                     sb.append("unbound");
                 } else if (isUnixSocket()) {
-                    sb.append(UnixDomainSockets.getRevealedLocalAddressAsString(addr));
-                } else {
-                    sb.append(Net.getRevealedLocalAddressAsString(addr));
+                    sb.append(addr.toString());
```

Comment:
Here's another case where the toString isn't needed.

Reply:
Fixed in 7b7f23c. Prior resolved conversations addressed in bc563ae.